app/src/main/java/com/tr4n/moviedb/ui/detail/MovieDetailFragment.kt

```diff
@@ -0,0 +1,53 @@
+package com.tr4n.moviedb.ui.detail
+
+import android.content.Context
+import android.os.Bundle
+import android.view.View
+import android.widget.Toast
+import androidx.navigation.navArgument
+import com.tr4n.moviedb.R
+import com.tr4n.moviedb.base.BaseFragment
+import com.tr4n.moviedb.utils.Constant
+import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.android.synthetic.main.fragment_movie_detail.*
+
+
+class MovieDetailFragment : BaseFragment(R.layout.fragment_movie_detail) {
+    private var movieId : Long = 0
+    override fun initData() {
+        //TODO("Not yet implemented")
+        Toast.makeText(activity, movieId.toString(), Toast.LENGTH_SHORT).show()
+    }
+
+    override fun listenEvents() {
+        buttonBackFromDetail.setOnClickListener {
+            activity?.supportFragmentManager?.popBackStack()
+        }
+    }
+
+    override fun observeData() {
+        //TODO("Not yet implemented")
+    }
+
+    override fun setupViews() {
+        //TODO("Not yet implemented")
+    }
+
+    override fun onAttach(context: Context) {
+        super.onAttach(context)
+        arguments?.getLong(Constant.ARGUMENT_MOVIE_ID_KEY)?.let {
+            movieId = it
+        }
+    }
+
+    companion object {
+        @JvmStatic
+        fun newInstance(movieId : Long) : MovieDetailFragment {
+            val movieDetailFragment = MovieDetailFragment()
+            val bundle = Bundle()
+            bundle.putLong(Constant.ARGUMENT_MOVIE_ID_KEY, movieId)
+            movieDetailFragment.arguments = bundle
+            return movieDetailFragment
+        }
+    }
```

Comment:
Lần sau viết theo mẫu a bảo nhé 
```suggestion
    companion object {
	
	     private const val BUNDLE_MOVIE_ID = "BUNDLE_MOVIE_ID"
		 
        fun newInstance(movieId : Long) : MovieDetailFragment {
            val movieDetailFragment = MovieDetailFragment().apply {
			    arguments = bundleOf(BUNDLE_MOVIE_ID to movieId)
			}
        }
    }
```